test/LEAF_Nexus_Tests/tests/api/controllers/GroupControllerTest.php

```diff
@@ -104,6 +104,23 @@ public function testSearchTag() : void
         $this->assertEquals('13', $group[0]['groupID']);
     }
 
+    /**
+     * Tests the `group/[digit]/tag` endpoint for deletion.
+     */
+    public function testDeleteTag() : void
+    {
+        //create a tag and check to make sure the it was successfully made
+        self::$client->postEncodedForm('?a=group/13/tag', array('tag' => "TESTTAG"));
+        $group = self::$client->get('?a=group/tag&tag=TESTTAG');
+        $this->assertEquals('TESTTAG', $group[0]['tag']);
```

Comment:
There should be a check to assure `$group` is not an empty array before trying to access an index of it.